content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -606,7 +607,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   filesystem walk for better performance and accuracy.
 - `LogarithmicScaleDown`: Enable semi-random selection of pods to evict on controller scaledown
   based on logarithmic bucketing of pod timestamps.
-- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for 
+- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for
```

Comment:
Ideally, don't let your editor touch this line.

Reply:
Sorry, my editor is configured to automatically format any file on save :upside_down_face: